### DISCUSSION TITLE
End Of Life and End Of Support for RKE release 1.23.7

### DISCUSSION THREAD

**Discussion by wahaajsiddiqui** [rancher/rke2#5081] [General]
Please share the details of End Of Life and End Of Support for RKE release 1.23.7. Can you please tell me whether Rancker officially supports RKE 1.23.7?

**Comment by brandond** [comment]
We only "support" the latest patch release on a given minor version. Support for the minor version follows the same terms as upstream.

You can find the end of life date for 1.23 at https://kubernetes.io/releases/patch-releases/#non-active-branch-history

**Reply by wahaajsiddiqui** [reply]
Thank you @brandond
